src/applications/vaos/new-appointment/redux/reducer.js

```diff
@@ -1016,6 +1017,53 @@ export default function formReducer(state = initialState, action) {
         },
       };
     }
+    case FORM_PAGE_COMMUNITY_CARE_PROVIDER_SELECTION_OPENED: {
```

Comment:
This is a copy of the prefs reducer, with one minor change to what field the title is set on below. I think this will probably diverge more from the old reducer, so I decided to copy it instead of adding a conditional to the other one.